build.cmd

```diff
@@ -1,6 +1,9 @@
 @if "%_echo%" neq "on" echo off
 setlocal EnableDelayedExpansion
 
+set cleanBuild=
+set _skipTests=false
+set _configurationgroup=Debug
```

Comment:
If we're going to override the default configuration of build.proj here, then we should also pass this ConfigurationGroup to the native build.